torch/optim/optimizer.py

```diff
@@ -27,6 +34,19 @@ def _use_grad(self, *args, **kwargs):
     return _use_grad
 
 
+def register_optimizer_step_pre_hook(hook: Callable[..., None]) -> RemovableHandle:
```

Comment:
We should remove these if they just return an error and do nothing.

Reply:
will do.